schema/sql/schema.sql

```diff
@@ -97,6 +97,14 @@ create table "ssh_public_key" (
   "uid" integer references passwd (uid) on delete cascade
 );
 
+create domain pgp_sha256_fingerprint bytea check (length(value) = 64);
+
+create table "openpgp_public_key" (
+  "uid" integer references passwd (uid) on delete cascade,
+  "key" bytea not null check (length(key) < 2097152), -- max 2M
```

Comment:
Can optimize this by only storing the ASCII armoured key, and generating `key` when we're exposing things through v1.openpgp_public_key.

Not a blocking issue.